pkgs/development/node-packages/aliases.nix

```
@@ -44,4 +44,5 @@ mapAliases {
   "@nestjs/cli" = pkgs.nest-cli; # Added 2023-05-06
   manta = pkgs.node-manta; # Added 2023-05-06
   triton = pkgs.triton; # Added 2023-05-06
+  eslint_d = pkgs.eslint_d;
```

```suggestion
  eslint_d = pkgs.eslint_d; # Added 2023-05-26
```
also we should sort the entry :)